androidapp/Crisislab/app/src/main/kotlin/com/example/crisislab/NotificationModule.kt

```diff
@@ -11,43 +11,47 @@ import androidx.core.app.NotificationCompat
 import androidx.core.app.NotificationManagerCompat
 
 object NotificationModule {
+
+    // Function to build a notification
     fun build(context: Context, title: String, message: String, bigText: String?): Notification {
-        // This intent is for when the user taps the notification, it opens the app.
+        // Intent to open the app when the user taps the notification
         val intent = Intent(context, MainActivity::class.java).apply {
             flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
         }
         val pendingIntent: PendingIntent = PendingIntent.getActivity(context, 0, intent, PendingIntent.FLAG_IMMUTABLE)
 
-        // This is for full screen notifications.
+        // Intent for full-screen notifications
         val fullScreenIntent = Intent(context, MainActivity::class.java)
-        val pendingFullScreenIntent = PendingIntent.getActivity(context, 0, fullScreenIntent,
-            PendingIntent.FLAG_IMMUTABLE)
+        val pendingFullScreenIntent = PendingIntent.getActivity(context, 0, fullScreenIntent, PendingIntent.FLAG_IMMUTABLE)
 
+        // Build and return the notification
         return NotificationCompat.Builder(context, "Main Channel ID")
             .setPriority(NotificationCompat.PRIORITY_DEFAULT)
             .setSmallIcon(R.drawable.ic_notif)
             .setContentTitle(title)
             .setContentText(message)
-            .setStyle(NotificationCompat.BigTextStyle().bigText(bigText))
-            .setPriority(NotificationCompat.PRIORITY_DEFAULT)
+            .setStyle(NotificationCompat.BigTextStyle().bigText(bigText))  // Set the style for the notification with big text
+            .setPriority(NotificationCompat.PRIORITY_DEFAULT)  // Set the priority of the notification
             .setContentIntent(pendingIntent)
             .setFullScreenIntent(pendingFullScreenIntent, true)
-            .setAutoCancel(true)
+            .setAutoCancel(true)  // Automatically remove the notification when the user taps it
             .setOngoing(true)
```

Comment:
What does this mean?

Reply:
I've asked 4 people and they all think this makes sense, so I'm just going to leave it like this